src/ol_concourse/pipelines/infrastructure/k8s_apps/meta.py

```diff
@@ -115,8 +122,29 @@ def meta_pipeline(app_names: list[str]) -> Pipeline:
 
 
 if __name__ == "__main__":
-    app_names = ["learn-ai", "unified-ecommerce", "mit-learn", "mitxonline"]
+    app_configs = {
+        "learn-ai": {
+            "repo_name": "learn-ai",
+            "dockerfile_path": "./Dockerfile",
+        },
+        "unified-ecommerce": {
+            "repo_name": "unified-ecommerce",
+            "dockerfile_path": "./Dockerfile",
+        },
+        "mit-learn": {
+            "repo_name": "mit-learn",
+            "dockerfile_path": "./Dockerfile",
```

Comment:
Does this want the `./`?

Reply:
It doesn't mind it and I kind of like being explicit. This is clearly requesting the dockerfile in the root of repository.